SimeonInstaller.ps1

```diff
@@ -1803,6 +1902,17 @@ CRLFOption=CRLFAlways
                 if (Test-Path "$_.yml") { Remove-Item "$_.yml" -Force -EA SilentlyContinue }
                 irm "https://raw.githubusercontent.com/simeoncloud/$ymlRepo/master/$_.yml" -OutFile "$_.yml"
             }
+
+            if (!$UseServiceAccount) {
```

Comment:
Should we remove the use cache parameter if using a service account?

Reply:
I suppose we could. Do you think we should do that?

Reply:
Yes

Reply:
Otherwise If you reinstall with use service account checked, it will still use the cache and not authenticate as the service account